blog/2024-06-18-async_register_static_path.md

```diff
@@ -0,0 +1,32 @@
+---
+author: J. Nick Koston
+authorURL: https://github.com/bdraco
+title: Making http path registration async safe with `async_register_static_path`
+---
+
+`hass.http.register_static_path` is deprecated because it does blocking I/O in the event loop, instead call `await hass.http.async_register_static_path([StaticPathConfig(url_path, path, cache_headers)])`
```

Comment:
```suggestion
`hass.http.register_static_path` is deprecated because it does blocking I/O in the event loop, instead call `await hass.http.async_register_static_paths([StaticPathConfig(url_path, path, cache_headers)])`
```